src/pages/grading/components/GradingDialog.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import {
+  Button,
+  DialogTitle,
+  Dialog,
+  Paper,
+  Divider,
+  DialogContentText,
+} from "@mui/material";
+import { styled } from "@mui/material/styles";
+
+const DialogContent = styled(DialogContentText)(({ theme }) => ({
+  margin: theme.spacing(1),
+  marginLeft: theme.spacing(3),
+  marginRight: theme.spacing(3),
+}));
+
+const Input = styled("input")({
+  display: "none",
+});
+
+const GradeButton = styled(Button)(({ theme }) => ({
+  margin: theme.spacing(1),
+}));
+
+const Footer = styled("div")({
+  float: "right",
+});
+
+interface GradingDialogProps {
+  open: boolean;
+  handleClose: () => void;
+}
+
+export function GradingDialog(props: GradingDialogProps) {
+  const { open, handleClose } = props;
+
+  const handleDialogSubmit = () => {
+    // TODO:
+    //  dont close before checking
+    //  if action was successful
+    // handleClose();
+  };
+
+  return (
+    <Dialog onClose={handleClose} open={open} maxWidth="md" fullWidth>
```

Comment:
This is a skeleton for the dialog of grading, I decided I will finish it next PR for time management reasons, please ignore that it is unfinished, unless it breaks something somewhere.